recipes/subunit/all/conanfile.py

```diff
@@ -0,0 +1,123 @@
+            self.build_requires("automake/1.16.2")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("{}-{}".format(self.name, self.version), self._source_subfolder)
+
+    @contextmanager
+    def _build_context(self):
+        env = {}
+        if self.settings.compiler == "Visual Studio":
+            with tools.vcvars(self.settings):
+                env.update({
+                    "AR": "{} lib".format(tools.unix_path(self.deps_user_info["automake"].ar_lib)),
+                    "CC": "{} cl -nologo".format(tools.unix_path(self.deps_user_info["automake"].compile)),
+                    "CXX": "{} cl -nologo".format(tools.unix_path(self.deps_user_info["automake"].compile)),
+                    "NM": "dumpbin -symbols",
+                    "OBJDUMP": ":",
+                    "RANLIB": ":",
+                    "STRIP": ":",
+                })
+                with tools.environment_append(env):
+                    yield
+        else:
+            with tools.environment_append(env):
+                yield
+
+    def _configure_autotools(self):
+        if self._autotools:
+            return self._autotools
+        self._autotools = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
+        self._autotools.libs = []
+        if self.settings.compiler == "Visual Studio":
+            self._autotools.flags.append("-FS")
+            self._autotools.cxx_flags.append("-EHsc")
+        yes_no = lambda v: "yes" if v else "no"
+        conf_args = [
+            "--enable-shared={}".format(yes_no(self.options.shared)),
+            "--enable-static={}".format(yes_no(not self.options.shared)),
+            "CHECK_CFLAGS=' '",
+            "CHECK_LIBS=' '",
+            "CPPUNIT_CFLAGS='{}'".format(" ".join("-I{}".format(inc) for inc in self.deps_cpp_info["cppunit"].include_paths).replace("\\", "/")),
+            "CPPUNIT_LIBS='{}'".format(" ".join(self.deps_cpp_info["cppunit"].libs)),
+        ]
+        self._autotools.configure(args=conf_args, configure_dir=self._source_subfolder)
+        return self._autotools
+
+    def build(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+        with self._build_context():
+            autotools = self._configure_autotools()
+            autotools.make()
+
+    def package(self):
+        self.copy("COPYING", src=self._source_subfolder, dst="licenses")
+        with self._build_context():
+            autotools = self._configure_autotools()
+            autotools.install()
+
+        os.unlink(os.path.join(self.package_folder, "lib", "libcppunit_subunit.la"))
+        os.unlink(os.path.join(self.package_folder, "lib", "libsubunit.la"))
+        for d in glob.glob(os.path.join(self.package_folder, "lib", "python*")):
+            tools.rmdir(d)
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
```

Comment:
```suggestion
        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
        tools.rmdir(os.path.join(self.package_folder, "Library"))
        tools.rmdir(os.path.join(self.package_folder, "lib", "perl5"))
```

Reply:
Applicable if a perl module is installed to the package directory.

Reply:
It created a `lib/x86_64-linux-gnu/perl` folder structure on a local docker instance.
In `package` all subdirectories of `lib` are now removed.